.github/workflows/rebuild-mpxv.yaml

```diff
@@ -35,6 +35,7 @@ jobs:
           . \
             envdir env.d snakemake notify_on_deploy \
               --configfiles config/config_mpxv.yaml config/nextstrain_automation.yaml \
+              --cores all \
```

Comment:
Hmm, I don't think we should use `--cores all` because of https://github.com/nextstrain/cli/issues/175. (I mentioned this earlier, but I might not have explained it well then.)

Reply:
I didn't use `all` whenever there was an explicit number given. I didn't know what to pass otherwise when `--cpus` wasn't passed.

What does `nextstrain build` use as `--cpus` by default? CLI `--help-all` said `None`? So if there's no machine up it won't run? Or will it be like 0.00001?

What would you put instead in case of no `--cpus`?

Reply:
It uses the default as configured in AWS Batch. That will vary for different organizations, but for us, it's currently 4.

For this case above, I would explicitly pass `--cpus 4` to `nextstrain build` to make the AWS Batch default explicit (until https://github.com/nextstrain/cli/issues/175 is fixed) and then also `--cores 4` to `snakemake` (until https://github.com/nextstrain/cli/issues/272 is fixed, probably via [first-class env var management support](https://github.com/tsibley/blab-standup/blob/master/2022-08-26.md)).

Reply:
I mean, ok, I guess, but I don't see how `--cores all` here is in any way better or quicker than `--cpus 4` + `--cores 4`.

Reply:
To make what I'd said earlier very clear: if we run concurrent workflows with small numbers of requested CPUs, they are likely to get packed onto the same machine, where overscheduling jobs by Snakemake is likely to make them all slower. Slow enough to notice? Maybe, maybe not. I don't know, as I rarely pay attention to our workflow execution times.